internal/backend/remote/backend_plan.go

```diff
@@ -315,6 +315,13 @@ in order to capture the filesystem context the remote workspace expects:
 		}
 	}
 
+	if len(op.Excludes) != 0 {
```

Comment:
This does the same thing for Excludes that is done for Includes on L311-314 above.  Is that right?  I would have expected some sort of "inverse" operation.

Reply:
I think I gave up with this change, since it's not possible to change `tfe.RunCreateOptions` to store `ExcludeAddrs`.

This is why I commented on the RFC that it might not be possible to support for certain backends (although I believe you went on to respond that the backend actually don't use the targets for anything).

When revisiting this / redoing this PR if the RFC were accepted, I would probably leave out all changes to this file if they're not needed.